### ISSUE TITLE
[BUG]: Unable to complete the operation because of an invalid property TargetIds .

### ISSUE THREAD

**Issue by noesberger** [dell/dellemc-openmanage-ansible-modules#496]
### Bug Description

We've installed the new OME Version 3.10 and are now getting an error in the module configuration_compliance_baseline
Unable to complete the operation because of an invalid property TargetIds .

The Problem is the following section in the code:
```
    pattern = re.compile(r'(1|2|3)\.(0|1|2|3|4)\.?')
    if pattern.match(ome_version):
        payload["TargetIds"] = noncomplaint_devices
    else:
        payload["DeviceIds"] = noncomplaint_devices
    return payload
```

the regex is now also matching for version 3.10 and is using the TargetIds instead of DeviceIds Option. So the payload is then wrong. We have to find a better way to differenciate the versions.

### Component or Module Name

configuration_compliance_baseline

### Ansible Version

 2.10.17

### Python Version

Python 3.9

### iDRAC/OME/OME-M version

Version 3.10.1 (Build 51)

### Operating System

Rhel 8

### Playbook Used

- name: "Apply OME configuration baseline"
  dellemc.openmanage.ome_configuration_compliance_baseline:
    hostname: "{{ ome_server }}"
    username: "{{ ome_username }}"
    password: "{{ ome_password }}"
    command: "remediate"
    names: "{{ baseline_name }}"
    device_ids: "{{ _device_id }}"
    validate_certs: false

### Logs

{
    "error_info": {
        "error": {
            "code": "Base.1.0.GeneralError",
            "message": "A general error has occurred. See ExtendedInfo for more information.",
            "@Message.ExtendedInfo": [
                {
                    "MessageId": "CGEN1015",
                    "RelatedProperties": [],
                    "Message": "Unable to complete the operation because of an invalid property TargetIds .",
                    "MessageArgs": [
                        "TargetIds"
                    ],
                    "Severity": "Critical",
                    "Resolution": "Verify the identified property is valid in the message and retry the operation with valid payload."
                }
            ]
        }
    },

### Steps to Reproduce

Just run the module with OME Version 3.10

### Expected Behavior

no error

### Actual Behavior

Unable to complete the operation because of an invalid property TargetIds .

**Comment by anupamaloke** [comment]
@noesberger, thank you for submitting this issue. Let us take a look and come back on this.

**Comment by sachin-apa** [comment]
@noesberger we are planning to use _LooseVersion_ library provided by the ansible to compare the version instead of regular expression and is supported from `ansible 2.12` version onwards.
As per the bug report I see that you are using `ansible 2.10.17` version which is pretty old. I hope you will be upgrading the ansible to the latest version.

Prerequisites defined for collection [here]( https://github.com/dell/dellemc-openmanage-ansible-modules#prerequisites )

**Comment by sachin-apa** [comment]
Fixed as part of the Releases 7.6.0 [PR ](https://github.com/dell/dellemc-openmanage-ansible-modules/pull/500)
